test/e2e/scheduling/predicates.go

```diff
@@ -658,8 +658,11 @@ var _ = SIGDescribe("SchedulerPredicates [Serial]", func() {
 		Testname: Scheduling, HostPort matching and HostIP and Protocol not-matching
 		Description: Pods with the same HostPort value MUST be able to be scheduled to the same node
 		if the HostIP or Protocol is different.
+		This test is marked LinuxOnly since hostNetwork is not supported on Windows
+		TODO: Add a new e2e test to scheduler which validates if hostPort is working and move this test to e2e/network
```

Comment:
Prob shud make a dig net issue to track this  good point

Reply:
Agreed, can you open an issue and update this comment to reference the issue?

Also please move TODO to new line so it doesn't get picked up as part of the description